core-plugin/build.gradle

```diff
@@ -0,0 +1,15 @@
+dependencies {
+    implementation "io.github.http-builder-ng:http-builder-ng-okhttp:${httpbuilderNgVersion}"
+    implementation "io.ratpack:ratpack-core:${ratpackVersion}"
```

Comment:
Using Ratpack to provide the local proxy server. WOuld prefer to use Micronaut instead, but so far has not been successful in getting that to go.